.github/workflows/chef-linear-notion-sync.yml

```
@@ -14,6 +14,8 @@ concurrency:
 
 jobs:
   sync:
+    # Public repos do not receive org secrets with visibility: private.
+    if: ${{ secrets.LINEAR_API_KEY != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)